### PR TITLE
Update gthesheep variant for snapchat-ads

### PR DESCRIPTION
**Pull request by GtheSheep**
## Checklist

- [x] Add/update the file in the appropriate folder (`/taps` or `/targets`). The name of the file should match the name of the tap. If there is already one, add a descriptor to the name such as `-search`.
- [x] Add/update the PNG logo image in `/assets/logos/<taps or targets>`. The image name must match the YAML file name.
- [x] Tag `@tayloramurphy` or `@pnadolny13` to flag it for review. Or post to the [#hub](https://meltano.slack.com/archives/C01UGBSJNG5) channel on Meltano slack.

@pnadolny13

## Reviewer Checklist

- [ ] Validate file against JSON Schema. https://www.jsonschema.net/home is an option.
- [ ] Build website locally and validate everything works.

### CODE REVIEW

**Comment by tayloramurphy** [comment]
@pnadolny13 will you review and then merge if all looks good?

File: _data/meltano/extractors/tap-snapchat-ads/gthesheep.yml [hunk 1]
```diff
@@ -53,6 +73,11 @@ settings_group_validation:
 - - refresh_token
   - client_secret
   - client_id
+  - start_date
+  - swipe_up_attribution_window
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-snapchat-ads/gthesheep.yml:77]: @GtheSheep are these required settings? I saw that the `--about` [output](https://github.com/meltano/hub/pull/1052#issuecomment-1331344621) from the CI bot didnt require them.

Comment (GtheSheep) [reply _data/meltano/extractors/tap-snapchat-ads/gthesheep.yml:77]: they aren't, is that what this `settings_group_validation` section is?

Comment (pnadolny13) [reply _data/meltano/extractors/tap-snapchat-ads/gthesheep.yml:77]: @GtheSheep yeah thats a list of required settings. Check out the [plugin definition syntax docs](https://docs.meltano.com/reference/plugin-definition-syntax#settings_group_validation) for more detail

Comment (GtheSheep) [reply _data/meltano/extractors/tap-snapchat-ads/gthesheep.yml]: Fair play, changed 👌

**Pull request merged**